Source/WebKit/GPUProcess/media/RemoteVideoFrameObjectHeap.mm

```diff
@@ -35,7 +35,10 @@
 
 RefPtr<WebCore::VideoFrame> RemoteVideoFrameObjectHeap::get(RemoteVideoFrameReadReference&& read)
 {
-    return m_heap.read(WTFMove(read), 0_s);
+    RefPtr result = m_heap.read(WTFMove(read), 5_s);
```

Comment:
This is probably the change that fixes the potential race, correct?
This can introduce potential small locks between the camera output thread and the local rendering thread.
Currently, these two are mostly independent with each other.

Reply:
No. It's just that if, say, RemoteGraphicsContextGL connection inside its work queue creates the video frame, it won't be available in WebRTC encoding thread until the RemoteGraphicsContextGL is done generating the video frame.